### ISSUE TITLE
could sandbox re-create local db tables when it detects arcfile changes?

### ISSUE THREAD

**Issue by filmaj** [architect/sandbox#7]
would be neat! we already have the filesystem watching set up for arc file changes to re-setup the HTTP routes; could we do the same for the db?

**Comment by brianleroux** [comment]
yes! we could totally shutdown and restart dynalite server. also would like to eventually give the ability to save state there.

**Comment by ryanblock** [comment]
Thought I created an issue for this – it's already on my list of things to do!

The approach I was going to take:
- Upon init, load the `arc` object into memory
- Any time the `.arc` file changes, diff it against the in-memory object, and
  - Reload any services that changed (if any)
  - Load the new `arc` object into memory for the next diff

(Right now it's dumb af, all we do is reload HTTP whenever the `.arc` file changes.)
